core/templates/components/cta/cta.twig

```diff
@@ -1,5 +1,31 @@
+{#
+/**
+ * @file
+ * CTA Component.
+ *
+ * Call to action component with clickable image and button.
+ * Variants:
+ *   - Default - Button at the bottom: With full-width button under the image
+ *   - Button in the center: With button in the center of the clickable image
+ *   - Button at the bottom with icon in the center: With full-width button under the image and an icon in the center of the image
+ *
+ * Available variables:
+ * - attributes: For additional HTML attributes not already provided.
+ * - modifier_class: Additional css classes to change look and behaviour.
+ * - src: The href property of the image.
```

Comment:
This is now icon_src. Thank you.

Reply:
I renamed the variables so they are a bit clearer.

Reply:
These are great.